connector/src/main/java/org/jboss/as/connector/subsystems/datasources/Constants.java

```diff
@@ -612,6 +611,7 @@ public void validateResolvedParameter(String parameterName, ModelNode value) thr
             EXCEPTION_SORTER_PROPERTIES,
             STALE_CONNECTION_CHECKER_PROPERTIES,
             VALID_CONNECTION_CHECKER_PROPERTIES, CONNECTION_LISTENER_PROPERTIES,
+            CONNECTION_PROPERTIES,
```

Comment:
It just moved from a `SimpleAttributeDefinition` array to a `PropertiesAttributeDefinition` array. Both are used in the constructor here https://github.com/wildfly/wildfly/pull/7682/files#diff-21c94a31536938f6500d62659d08d703R43. tl;dr the type changed to better reflect the type being represented.